python/ql/src/semmle/python/ApiGraphs.qll

```diff
@@ -351,20 +351,95 @@ module API {
 
     private import semmle.python.types.Builtins as Builtins
 
+    /** Returns the names of known built-ins. */
+    private string builtin_name() {
+      // Built-in functions and exceptions shared between Python 2 and 3
+      result in [
+          "abs", "all", "any", "bin", "bool", "bytearray", "callable", "chr", "classmethod",
+          "compile", "complex", "delattr", "dict", "dir", "divmod", "enumerate", "eval", "filter",
+          "float", "format", "frozenset", "getattr", "globals", "hasattr", "hash", "help", "hex",
+          "id", "input", "int", "isinstance", "issubclass", "iter", "len", "list", "locals", "map",
+          "max", "memoryview", "min", "next", "object", "oct", "open", "ord", "pow", "print",
+          "property", "range", "repr", "reversed", "round", "set", "setattr", "slice", "sorted",
+          "staticmethod", "str", "sum", "super", "tuple", "type", "vars", "zip", "__import__",
```

Comment:
Thanks for explaining :pray: sounds very reasonable to me 

> I should probably add a comment to this effect, in case we wish to redo this in the future.

That sounds very nice :+1: